src/app/js/formats/streamgraph/index.js

```diff
@@ -0,0 +1,9 @@
+import Component from './Streamgraph';
+import DefaultFormat from '../DefaultFormat';
+import Icon from './StreamgraphIcon';
+
+export default {
+    ...DefaultFormat,
+    Component,
+    Icon,
+};
```

Comment:
Please let a line at the end of the file.

Reply:
:+1: